pipelines/solSysPipe.yaml

```diff
@@ -5,9 +5,9 @@ tasks:
   consolidateSsp:
     class: lsst.solsys.pipe.consolidateSspTables.ConsolidateSspTablesTask
     config:
-      connections.diaSourceInputName: dia_source_detector
-      connections.visitSummaryInputName: visit_summary
-  makeTracklets: lsst.solsys.pipe.makeTracklets.makeTrackletsTask
-  heliolinc: lsst.solsys.pipe.heliolinc.heliolincTask
-  linkPurify: lsst.solsys.pipe.linkPurify.linkPurifyTask
-  submitToMPC: lsst.solsys.pipe.submitToMPC.submitToMPCTask
+      connections.diaSourceInputName: "dia_source_detector"
+      connections.visitSummaryInputName: "visit_summary"
```

Comment:
I don't think we need quotations around `dia_source_detector` and `visit_summary`. They'll be parsed as strings.